src/libsync/bandwidthmanager.h

```diff
@@ -87,14 +87,14 @@ public slots:
     qint64 _relativeUploadLimitProgressAtMeasuringRestart;
     qint64 _currentUploadLimit;
 
-    std::list<GETJob *> _downloadJobList;
+    std::list<GETFileJob *> _downloadJobList;
```

Comment:
Maybe we should call the class `GetFileJob` to use the naming pattern used by the other classes.

Reply:
The class had that name for 10 years